### DISCUSSION TITLE
Any ideas how Sunshine consumes power/battery life in the background?

### DISCUSSION THREAD

**Discussion by lyndonguitar** [LizardByte/community#233] [Sunshine] [answered]
I was thinking of running this on an ROG Ally (soon) so I can easily do multiplayer with my Steam Deck running moonlight, however, my  concern is the potential increased battery consumption with running Sunshine in the background

If yes, is there a way to manually activate/deactivate sunshine at will so the server will only run at will?

**Comment by ld892012** [comment]
Hello
You can set Sunshine starts only on-demand by typing this in a Powershell console (admin) :
`Set-Service -Name sunshinesvc -StartupType Manual`

After that, Sunshine not start automatically. If you want to start it you can type this in a Powershell console (admin) : 
`Start-Service -Name "sunshinesvc"`

To open a Powershell console you just have to do a right click to the Windows start menu logo and select `Terminal (admin)` or `Powershell (admin)`

**Comment by cgutman** [comment, marked as answer]
Sunshine should have no performance impact in the background when you're not streaming. You can check CPU/GPU usage in Task Manager to confirm.

If you see an impact when not streaming, file a bug and we can investigate.

**Comment by ld892012** [comment]
In my computer, Sunshine only consumes 10mb of RAM (when GeForce Experience eating 140mb) ^^
